src/Assignment/DriverToBus/controllers/AssignDrivertoBus.controller.js

```diff
@@ -23,10 +22,11 @@ const AssignDriverToBus = async (req, res) => {
         status: "fail",
         message: "Driver is already assigned to a Bus",
       });
-    } else {
-      user.isAssigned = true;
-      await user.save();
-    }
+    } 
+    // else {
+    //   user.isAssigned = true;
+    //   await user.save();
+    // }
```

Comment:
Remove any un needed, commented-out code

Reply:
this was not my codes it was a bug and I resolved it . But anyway I will remove it